cmd/srcd/cmd/errors.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/src-d/engine/docker"
+
+	"github.com/sirupsen/logrus"
+)
+
+// fatal converts known errors to human friendly message and logs it with fatal level
+func fatal(err error, format string, args ...interface{}) {
+	msg := fmt.Sprintf(format, args...)
+	err = docker.ParseErr(err)
+	errString := err.Error()
+
+	switch e := err.(type) {
+	case *docker.ContainerBindErr:
+		// TODO(max): instead of placeholders we can actually ask daemon for current config if we would have such API
+		confFile := "path/to/config.yml"
+		workdir := "[workdir]"
+
+		errString = "Port " + e.Port + " is already allocated.\n" +
```

Comment:
Maybe we could use the default path for the config file ($HOME/.srcd/config.yml).
Also I think we should link to https://docs.sourced.tech/engine/learn-more/commands#srcd, otherwise as a user you don't have a way to know what you are supposed to write in the file.

Reply:
I'd avoid linking external resources in go files if we don't have a way to validate links integrity.
I agree on mentioning  default place to store it.
What I'd really really really like, is a command to generate that file. Wouldn't be great? 😅